src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.scss

```diff
@@ -7,30 +16,32 @@
 .modal-header {
   border-bottom: none;
 }
+.modal-header .close {
+  color: #fff;
+  font-size: 2em;
+}
 .modal-body {
   padding-left: 80px;
   padding-right: 80px;
 }
-.modal-footer {
-  border-top: none;
-}
-h2 {
-  font-size: 3em;
-}
 .ceph-logo {
-  width: 10%;
-}
-.list-unstyled li {
-  margin-top: 5px;
+  width: 25%;
+  margin-bottom: 30px;
+  object-position: -20px 0;
 }
 .modal-footer {
+  display: block;
+  border-top: none;
   padding: 15px 80px 35px 80px;
 }
-.modal-body {
-  margin-top: 0;
-  padding-top: 0;
-
-  h2 {
-    margin-top: 0;
-  }
+.about-container dl {
+  @include make-row();
+  margin:inherit;
+}
+.about-container dt {
+  @include make-col(4);
+  font-weight: $font-weight-bold;
+}
+.about-container dd {
+  @include make-col(8);
```

Comment:
Minor suggestion: you can use some of extensions [SCSS provide (like nesting)](https://sass-lang.com/guide) over CSS:
```scss
.about-container {
  dl { }
  dt { }
  dd { }
}
```

Reply:
Thanks @epuertat. Here is a screenshot of the latest changes
![Screenshot from 2020-04-07 16-15-03](https://user-images.githubusercontent.com/15856785/78686688-41836800-78eb-11ea-8b8b-587f9dc5c38c.png)

Reply:
> Minor suggestion: you can use some of extensions [SCSS provide (like nesting)](https://sass-lang.com/guide) over CSS:
> 
> ```css-scss
> .about-container {
>   dl { }
>   dt { }
>   dd { }
> }
Ok , this sure looks better.